libs/common-utils/src/environments.ts

```diff
@@ -3,9 +3,11 @@ import { registerOnWindow } from './misc'
 const DEFAULT_ENVIRONMENTS_REGEX: Record<EnvironmentName, string> = {
   local: '^(:?localhost:\\d{2,5}|(?:127|192)(?:\\.[0-9]{1,3}){3})',
   pr: '^((?:explorer|swap)-dev-git-[\\w\\d-]+|swap-\\w{9}-)cowswap-dev\\.vercel\\.app',
-  development: '^(dev.swap.cow.fi|swap-develop.vercel.app)',
-  staging: '^(staging.swap.cow.fi|swap-staging.vercel.app)',
-  production: '^(swap.cow.fi|swap-prod.vercel.app)$',
+  development:
+    '^(dev.swap.cow.fi|dev.explorer.cow.fi|swap-develop.vercel.app|explorer-dev-git-develop-cowswap-dev.vercel.app)',
```

Comment:
The last explorer domain doesn't look right. I mean, it works, but is it pointing to dev?
Anyway, here are the vercel domains for dev "prod":

<img width="347" height="176" alt="image" src="https://github.com/user-attachments/assets/5952255c-ab93-4627-b86a-bda4e0c46e7c" />

Swap looks good:
<img width="349" height="168" alt="image" src="https://github.com/user-attachments/assets/b83afe3c-ea18-498f-b7b2-5b089ec92144" />